salt/metalk8s/orchestrate/backup/files/job.yaml.j2

```diff
@@ -22,7 +22,10 @@ spec:
       containers:
       - name: backup-replication
         image: {{ image }}
-        command:
+        command: ["/bin/sh", "-c"]
+        args:
+          - ls -t -1 /var/lib/metalk8s/backups/*.tar.gz | tail -n+6 | xargs rm -f
```

Comment:
could this be in the pillar ? I can't find it

Reply:
You mean the `path` ? 
Here you are in a Pod and has you can see below
```yaml
        volumeMounts:
        - name: backups
          mountPath: /backups
```
The backups are mounted in `/backups` in the Pod

Reply:
that makes it a lot easier, thanks !